alicloud/table_alicloud_action_trail.go

```diff
@@ -31,11 +31,6 @@ func tableAlicloudActionTrail(ctx context.Context) *plugin.Table {
 				Description: "The name of the trail.",
 				Type:        proto.ColumnType_STRING,
 			},
-			{
-				Name:        "mns_topic_arn",
```

Comment:
Why is this table having a column removed in this PR?

Reply:
removed as discussed